### ISSUE TITLE
Style lint deprecation

### ISSUE THREAD

**Issue by marekxhtml** [xfiveco/generator-chisel#162]
Some of the style lint settings will be deprecated

```
Deprecation Warning: 'declaration-block-no-ignored-properties' has been deprecated and in 8.0 will be removed. See: https://stylelint.io/user-guide/rules/declaration-block-no-ignored-properties/
Deprecation Warning: 'media-feature-no-missing-punctuation' has been deprecated and in 8.0 will be removed. See: https://stylelint.io/user-guide/rules/media-feature-no-missing-punctuation/
Deprecation Warning: 'rule-nested-empty-line-before' has been deprecated and in 8.0 will be removed. Instead use 'rule-empty-line-before'. See: https://stylelint.io/user-guide/rules/rule-nested-empty-line-before/
Deprecation Warning: 'rule-non-nested-empty-line-before' has been deprecated and in 8.0 will be removed. Instead use 'rule-empty-line-before'. See: https://stylelint.io/user-guide/rules/rule-non-nested-empty-line-before/
```

**Comment by thymikee** [comment]
@marekxhtml will you send a PR for this? :)

**Comment by luboskmetko** [comment]
@marekxhtml are you using the latest version of the generator? Because this should be already addressed in the 0.6.0 release https://github.com/xfiveco/generator-chisel/commit/0ef3650f0470fa91e85b71f076f077319de78f3b#/diff-8507458e5cdfaf5aa5da8ecc7fdaaee8

**Comment by marekxhtml** [comment]
@luboskmetko I thought that I had the latest version, but it looks like that I have some issues with the npm update. Thanks for the info!
